src/blender_mcp/server.py

```
@@ -692,6 +692,148 @@ def import_generated_asset(
+        result = blender.send_command("create_hunyuan_job", {
+            "text_prompt": None,
+            "image": input_image_url,
+        })
+        if "JobId" in result.get("Response", {}):
+            job_id = result["Response"]["JobId"]
+            formatted_job_id = f"job_{job_id}"
+            return json.dumps({
+                "job_id": formatted_job_id,
+            })
+        else:
+            return json.dumps(result)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)